libs/npm/core/src/components/table/FlexboxTable.tsx

```diff
@@ -0,0 +1,193 @@
+import React from 'react';
+
+import { Text } from '../form';
+import { ITableProps, SortFlag, TablePager, useTable } from '.';
+import * as styled from './styled';
+
+export const FlexboxTable = <T extends object>({
+  rowId,
+  columns,
+  data,
+  ...rest
+}: ITableProps<T>) => {
+  const table = useTable({
+    rowId,
+    columns,
+    data,
+    options: {
+      isMulti: rest.isMulti,
```

Comment:
I'd like to find a better way to do this as currently it require manually passing the values down.